pep-9999.rst

```diff
@@ -47,13 +47,57 @@ required to interpret all standard constructs, and additionally a type checker
 can support extra constructs not described in this PEP. All deviations from
 this PEP should be documented.
 
+Syntax and Directory Structure
+==============================
+
+Type stubs are syntactically valid Python files with a ``.pyi`` suffix.
+Type checkers are expected to support the syntax of the latest Python
+version. Type stubs should be parseable with the latest released Python
+version, but also support the previous version up until six months after
+the release of the current version. Older Python versions do not need to be
+supported, even if the implementation supports older versions or the
+type checker is run with an older version.
+
+For example, it is recommended to use PEP 526 [#pep526]_ variable
+annotations, introduced in Python 3.6, in all stubs, even for Python 2-only
+modules.
+
+Type stub files can either live next to their respective implementation
+files or in a separate directory::
+
+    pkg
+        __init__.py
+        __init__.pyi
+        mod.py
+        mod.pyi
+
+    pkg
+        __init__.py
+        mod.py
+    stubs
```

Comment:
I think this is supposed to be `pkg-stubs`, according to the naming scheme in PEP 561.

Reply:
Oh, my example is confusing. I was trying to give an example for stubs for a stand-alone program, but of course this structure will not work when distributing the type hints via pypi. On the other hand, the `pkg-stubs` naming is for stubs-only packages. Maybe have three examples with better explanations:

* Same directory (for distributing as part of the package)
* `pkg-stubs` (for distributing as part of the package or as stand-alone package)
* `stubs/pkg` (for stand-alone apps, where you won't distribute the stubs)